server.js

```diff
@@ -9,11 +9,14 @@ import Iso from 'iso';
 import webpack from 'webpack';
 import cookieParser from 'cookie-parser';
 import bodyParser from 'body-parser';
+import Store from '@Store';
+import { loadDataForRoutes } from '@dataLoaderUtil';
 
 import alt from './src/app/alt';
 import appConfig from './src/app/data/appConfig';
 import webpackConfig from './webpack.config';
 import apiRoutes from './src/server/ApiRoutes/ApiRoutes';
+import RouteMethods from './src/server/ApiRoutes/RouteMethods';
```

Comment:
`RouteMethods` is not a class but an object. It should be lowercase.

Reply:
Fixed